src/main/java/helper/DataHelper.java

```diff
@@ -0,0 +1,20 @@
+package helper;
+
+import com.github.javafaker.Faker;
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class DataHelper {
+    private static final Faker faker = new Faker();
+
+    public static String getRandomEmail() {
+        return faker.letterify("??????????@gmail.com");
+    }
+
+    public static String getRandomAlphanumeric(int number) {
+        return RandomStringUtils.randomAlphanumeric(number);
```

Comment:
can't you use faker instead?

Reply:
yes, but this makes code shorter

Reply:
but I can change to use faker if you want

Reply:
I changed to use faker. Please check again